test/dbus/lvmdbustest.py

```diff
@@ -68,6 +68,42 @@ def lv_n(suffix=None):
 	return g_prefix + rs(8, s)
 
 
+def is_nested_pv(pv_name):
+	return pv_name.count('/') == 3
```

Comment:
Regardless of nested depth, lvm always has the device path as `/dev/vg_name/lv_name`  I've tested the nesting to 32 and the current test does 5.